tests/UI/pages/BO/international/localization/index.js

```diff
@@ -30,10 +40,10 @@ class Localization extends LocalizationBasePage {
   /* Methods */
   /**
    * Import a localization pack
-   * @param page
-   * @param country
-   * @param contentToImport
-   * @param downloadPackData
+   * @param page {Page} Browser tab
+   * @param country {string} Country to select
+   * @param contentToImport {contentToImport} Data of content to import to choose
```

Comment:
What is a contenToImport type?

If it's an object, document it as `{Object}`, same for others types such as zoneData

```suggestion
   * @param contentToImport {Object} Data of content to import to choose
```